setup.py

```diff
@@ -34,9 +34,9 @@
         'django_prices_openexchangerates.templatetags'],
     include_package_data=True,
     classifiers=CLASSIFIERS,
-    install_requires=['Django>=1.4', 'django-prices>=0.6.1', 'prices>=0.5.2'],
+    install_requires=['Django>=1.4', 'pytest-runner', 'django-prices>=0.6.1', 'prices>=1.0.0-beta'],
```

Comment:
What is `pytest-runner` doing here?

Reply:
This is still in progress, with @elwoodxblues we had a problem to run tests. For my environment it is still not resolved. Adding pytest-runner solved problem for @elwoodxblues. I am investigating this problem right now.

Reply:
We were checking if `pytest-runner` would help us locally, but this change shouldn't be committed.

Reply:
ok i will delete this